db/migrate/20130827094554_alchemy_two_point_six.rb

```diff
@@ -310,6 +310,88 @@ def up
         t.string "name"
       end
     end
+  end
+
+  def down
+    if table_exists?('alchemy_attachments')
+      drop_table "alchemy_attachments"
+    end
+
+    if table_exists?('alchemy_cells')
+      drop_table "alchemy_cells"
+    end
+
+    if table_exists?('alchemy_contents')
+      drop_table "alchemy_contents"
+    end
+
+    if table_exists?('alchemy_elements')
+      drop_table "alchemy_elements"
+    end
+
+    if table_exists?('alchemy_elements_alchemy_pages')
+      drop_table "alchemy_elements_alchemy_pages"
+    end
+
+    if table_exists?('alchemy_essence_booleans')
+      drop_table "alchemy_essence_booleans"
+    end
+
+    if table_exists?('alchemy_essence_dates')
+      drop_table "alchemy_essence_dates"
+    end
+
+    if table_exists?('alchemy_essence_files')
+      drop_table "alchemy_essence_files"
+    end
+
+    if table_exists?('alchemy_essence_htmls')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.